package.json

```diff
@@ -11,8 +11,9 @@
     "e2e": "node test/e2e/runner.js",
     "test": "npm run unit && npm run e2e",
     "lint": "eslint --ext .js,.vue src test/unit test/e2e/specs",
-    "build": "node build/build.js"
+    "prepare": "node build/build.js"
   },
+  "files": [ "dist/" ],
```

Comment:
@romkaspb I changed this to match `disciplina-faucet-frontend` because otherwise, the `dist` directory wasn't generated. I'm not sure why, but I'm guessing because it internally uses the build command which was overridden. Could you comment if this change is ok, or otherwise why this repo differs from the other?